Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -26,16 +26,17 @@
 
 WI.InlineSwatch = class InlineSwatch extends WI.Object
 {
-    constructor(type, value, {readOnly, preventChangingColorFormats} = {})
+    constructor(type, value, {readOnly, preventChangingColorFormats, variableType, tooltip, delegate} = {})
     {
         super();
 
         this._type = type;
+        this._variableType = variableType;
```

Comment:
this should be `this._variableType = variableType || null;` in case the caller doesn't provide a `variableType`